app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, source_file, copied_file = command.split()
+
+    if cp_command != "cp" or source_file == copied_file:
+        return
+
+    with open(source_file, "r") as source, open(copied_file, "w") as copied:
```

Comment:
The variable names `source` and `copied` are a bit vague. Consider using more descriptive names like `source_file` and `destination_file`.